client/rest-high-level/src/test/java/org/elasticsearch/client/CrudIT.java

```diff
@@ -65,6 +64,11 @@
     public void testDelete() throws IOException {
         {
             // Testing non existing document
+            final XContentType xContentType = randomFrom(XContentType.values());
```

Comment:
I could not figure out how else to explicitly create an index in the REST client... Do you have a suggestion?